src/main/java/DialogBox.java

```diff
@@ -0,0 +1,72 @@
+import java.awt.*;
```

Comment:
Should not import all as * as stated in the coding violation. Also, all classes in should in a package so the package statement is missing here.